app.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/takeaway.rb'
+require './lib/menu.rb'
+require './lib/order.rb'
+
+mochi_shop = Takeaway.new(menu: Menu.new)
```

Comment:
I think this could be easily turned into a feature test, by adding some expectations. Sam Morgan's video is very good for this.